pgjdbc/build.gradle.kts

```diff
@@ -224,6 +224,8 @@ tasks.configureEach<Jar> {
 tasks.shadowJar {
     configurations = listOf(shaded)
     exclude("META-INF/maven/**")
+    exclude("META-INF/versions/**")
+    exclude("META-INF/services/com.ongres.stringprep.Profile") // TODO
```

Comment:
Do you mean the service should be renamed when shading?

Reply:
The service file should be removed, is not needed here, this is mostly needed when stringprep is used independently, and want stringprep profile loaded dynamically (not the case for SCRAM dependency).

In other words for the shaded dependency this file should be removed to avoid conflicts with an application that explicitly use stringprep dependency.